docs/conf.py

```diff
@@ -200,7 +200,16 @@
 # resolve.
 
 nitpicky = True
-nitpick_ignore = []
+# This list is from https://github.com/numpy/numpydoc/issues/275
+nitpick_ignore = [
```

Comment:
I think we should add them to nitpick_exceptions; I found it needs to be without the quotes:
```
# sphinx doing strange things with dict methods
py:class None.  Remove all items from D.
py:class a set-like object providing a view on D's items
py:class a set-like object providing a view on D's keys
py:class v, remove specified key and return the corresponding value.
py:class None.  Update D from dict/iterable E and F.
py:class an object providing a view on D's values
py:class a shallow copy of D
```

Reply:
Done. Thanks!